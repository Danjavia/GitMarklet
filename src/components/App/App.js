/**
 * External Resources
 **/
import React from 'react';

/**
 * Internal Resources
 **/
import './App.css';

/**
 * App Component Declaration
 **/
export default ({ children }) => {
  return (
    <div className="container">
      {children}
    </div>
  );
};