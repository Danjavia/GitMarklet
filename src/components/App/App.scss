body {
  min-height: 100vh;

  .container {
    min-height: 93vh;
  }

  .ant-tag {
    line-height: 14px;
    font-size: 10px;
    height: 16px;
    margin-right: 0!important;
  }

  .flex {
    display: flex;
    height: 50vh;
    justify-content: center;
    align-items: center;
  }
}