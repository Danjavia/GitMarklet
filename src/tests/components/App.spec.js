import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from '../../components/App/App';

/** @test {AppComponent} */
describe('AppComponent', () => {

    /** @test {AppComponent#render-snapshot} */
    it('renders a snapshot', () => {
        const tree = renderer.create(<App />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    /** @test {AppComponent#render-without-crashing} */
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<App />, div);
    });
});
