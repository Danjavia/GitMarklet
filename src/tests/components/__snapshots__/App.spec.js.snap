exports[`AppComponent renders a snapshot 1`] = `
<div
  className="container" />
`;
